service/storage_service.cc

```diff
@@ -2024,6 +2034,9 @@ future<> storage_service::join_token_ring(sharded<db::system_distributed_keyspac
 
     set_mode(mode::NORMAL);
 
+    utils::get_local_injector().inject("stop_after_setting_mode_to_normal",
```

Comment:
Two injections with the same name?

Reply:
One for Raft and another one for gossip.

Reply:
No, there is how it works:

https://github.com/scylladb/scylladb/blob/15f8046fcb771e496d7e0db36ff2051c100d2593/utils/error_injection.hh#L450-L474

In case of this error injection it's not possible to reach both points in one run anyway.

Reply:
I'd still disambiguate this @enaydanov . Why do you need two separate injections, imagine the maintenance of this test, one would have to wonder which injection worked this time when debugging this.